### PR TITLE
Update of _posts/2014-07-29-installing.md

### PR DESCRIPTION
**Pull request by stefaweb**
Changes for 1.2.0-RC5

### CODE REVIEW

File: _posts/2014-07-29-installing.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-date: 2014-07-29 18:35:27
+date: 2017-02-21
 order: 1
 ---
 
 ## Installation instructions
 
 MailWatch for MailScanner is developed on Debian 7 & Ubuntu 12.04, so these docs will reflect this and I will make note on anything that will be required to run on other distro's or operating systems.
 
+If you're upgrading from a previous version, read `UPGRADING`.
```
Comment (endelwar) [inline _posts/2014-07-29-installing.md:13]: needs to be linked to `_posts/2017-02-03-upgrading.md`

File: _posts/2014-07-29-installing.md [hunk 2]
```diff
@@ -263,51 +269,56 @@ Point your browser to http://your-server-hostname/mailscanner/ - you should be p
 - Update the GeoIP database
   Click on the 'Other' menu and select 'Update GeoIP database' and click 'Run Now'.
 
-### Optional steps for Sendmail
+### Optional steps for Exil or Sendmail
```
Comment (endelwar) [inline _posts/2014-07-29-installing.md:272]: Typo

File: _posts/2014-07-29-installing.md [hunk 3]
```diff
@@ -263,51 +269,56 @@ Point your browser to http://your-server-hostname/mailscanner/ - you should be p
...
 
-Edit `tools/Sendmail_queue/mailq.php` to change the require line to point to the location of `functions.php`; copy `mailq.php` file to `/usr/local/bin` and setup the cronjob:
+copy the `tools/Sendmail-Exim_queue/mailwatch_sendmail_queue.php` file in `/usr/local/bin/`
+
+Verify if the file `mailwatch_sendmail_queue.php` is executable. If not, use "chmod +x" to correct this problem.
+
+setup the cronjob:
 
 ```shell
- $ cp tools/Sendmail_queue/mailq.php /usr/local/bin
```
Comment (endelwar) [inline _posts/2014-07-29-installing.md:275]: a copy/paste `cp` command is useful when following documentation, readd this line (with correct path)

File: _posts/2014-07-29-installing.md [hunk 4]
```diff
@@ -263,51 +269,56 @@ Point your browser to http://your-server-hostname/mailscanner/ - you should be p
...
-
- 0-59 * * * * 	/usr/local/bin/mailq.php
+```
+```
+# Run sendmail_queue.php each minute
+0-59 * * * * 	/usr/local/bin/mailwatch_sendmail_queue.php
 ```
 
-Note: `mailq.php` re-creates all entries on each run, so for busy sites you will probably want to change this to run every 5 minutes or greater.
+Note: mailwatch_sendmail_queue.php re-creates all entries on each run, so for busy sites you will probably want to change this to run every 5 minutes or greater.
```
Comment (endelwar) [inline _posts/2014-07-29-installing.md:292]: all files and command are enclosed in backtick, put them where appropriate

File: _posts/2014-07-29-installing.md [hunk 5]
```diff
@@ -329,7 +340,7 @@ Change file permissions so that we can update the rules and change group and rul
  $ chmod g+rw /etc/MailScanner/rules/*.rules
 ```
 
-See also the INSTALL docs in `tools/MailScanner_rule_editor` and `tools/Cron_jobs` directories.
+See also the INSTALL docs in `tools/MailScanner_rule_editor/INSTALL` and in `tools/Cron_jobs/INSTALL` directories.
```
Comment (endelwar) [inline _posts/2014-07-29-installing.md:343]: misleading phrase: is `tools/MailScanner_rule_editor/INSTALL` a directory? no it isn't but seems like it is.
Fow now revert to old version; before 1.2.0 all `INSTALL` files need to be converted to online documentation and removed from MailWatch repository

**Comment by stefaweb** [comment]
I delete this PR an start a new fresh PR.

**Pull request closed without merging**